docs/main/Solutioning/openebs-on-kubernetes-platforms/openshift.md

```diff
@@ -0,0 +1,563 @@
+---
+id: openshift
+title: Replicated PV Mayastor Installation on OpenShift
```

Comment:
Is this installation of only replicated pv mayastor? Do we want to do this or should it be OpenEBS umbrella?

Reply:
Usually, we have mentioned Mayastor in the other guides as well.

Reply:
Should we be doing that? If we keep it engine generic then it works for every one.